src/worker/cht-conf-worker.ts

```diff
@@ -103,19 +105,23 @@ export class MoveContactWorker {
     }
   }
 
+  private static fetchMonitoringApi(instanceUrl: string) {
+    return axios.get(`${instanceUrl}/api/v2/monitoring`);
+  }
+
   private static async moveContact(job: Job): Promise<JobResult> {
```

Comment:
suggestion: should we rename this function a well to align with the refactored generalization ?

Reply:
yep! `processChtConfJob`